### ISSUE TITLE
Release v1.x.z

### ISSUE THREAD

**Issue by Lyvsain** [Tencent/rapidjson#2021]
I've seen a lot of people asking for eat but I'll do it again : @miloyip can you do a newofficial release so that we can add the version on Conan.io ?

**Comment by Lyvsain** [comment]
Tracks #1006

**Comment by paulocoutinhox** [comment]
+1

**Comment by Spixmaster** [comment]
I agree.

**Comment by sirus20x6** [comment]
yes please. it looks like there have been thousands of commits since 2016, and some look to fix important things like use after free. please make a new release.

**Comment by brjoha** [comment]
There are a couple of viable alternatives to RapidJSON these days too.

Boost.JSON is comparable with RapidJSON in both parsing and serialization performance... https://www.boost.org/doc/libs/1_79_0/libs/json/doc/html/json/benchmarks.html

For jaw-dropping parsing performance, there's SIMDJSON... https://github.com/simdjson/simdjson#performance-results

**Comment by enali** [comment]
+1

**Comment by paulocoutinhox** [comment]
+1
